combat-trainer.lic

```diff
@@ -2762,6 +2763,7 @@ class TrainerProcess
     return if game_state.npcs.empty?
     return if fail_count > @analyze_retry_count
     return if DRSkill.getxp('Tactics') >= @combat_training_abilities_target
+    return unless game_state.is_engaging_allowed?
```

Comment:
I'm not super familiar with Innocence (I just read its entry on elanthipedia), so maybe this is a dumb question. Why would someone using Innocence also be using Analyze?

Reply:
Misconfiguration? It should be safe to return here regardless, as the two are mutually exclusive.